public/counter.html

```diff
@@ -1,51 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../assets/css/counter.css" />
     <title>Counter</title>
-    <style>
-        body{
-            text-align: center;
-        }
-        
-        h1{
-            font-size: 4rem;
-            margin-bottom: 1%;
-        }
-
-        div{
-            margin-bottom: 1%;
-        }
-
-        span{
-            font-size: 5rem;
-        }
-        
-        .container{
-            margin-top: 10%;
-        }
-
-        button{
-            padding: .5rem;
-            margin: .5rem;
-            border-radius: 4%;
-        }
-    </style>
-</head>
-<body>
+  </head>
+  <body>
     <div class="container">
-        <h1>Counter</h1>
-        <div>
-            <span id="count">0</span>
-        </div>
-        <div>
-            <button id="decrease">Decrease Count</button>
-            <button id="reset">Reset</button>
-            <button id="increase">Increase Count</button>
-        </div>
+      <h1>Counter</h1>
+      <div>
+        <span id="count">0</span>
+      </div>
+      <div>
```

Comment:
What is the need of giving different class to this button ? You can use "id" for that purpose. And always try to give meaning full name.

Reply:
Thanks for the suggestion I will work on it and will commit those changes shortly with Screenshot .

Reply:
![Counter - Google Chrome 08-02-2021 21_01_09](https://user-images.githubusercontent.com/64947831/107241335-e8c3b500-6a50-11eb-911b-6ff424e4f2db.png)

Reply:
This is the UI as same as You have done with calculator